tests/lambda_endpoints/test_indexer.py

```diff
@@ -1,128 +1,18 @@
-import os
-import time
 import unittest
 
-import boto3
 import pytest
-from botocore.exceptions import ClientError
-from opensearchpy import RequestsHttpConnection
-
-from sds_data_manager.lambda_code.SDSCode import indexer
-from sds_data_manager.lambda_code.SDSCode.opensearch_utils.action import Action
-from sds_data_manager.lambda_code.SDSCode.opensearch_utils.client import Client
-from sds_data_manager.lambda_code.SDSCode.opensearch_utils.document import Document
-from sds_data_manager.lambda_code.SDSCode.opensearch_utils.index import Index
 
 
 @pytest.mark.network()
 class TestIndexer(unittest.TestCase):
```

Comment:
Can probably remove this entire file for now too? We have been moving over to the pytest fixture-based approach as well, so might want to change this file over to that by starting fresh.